pkg/thing/mocks/publisher.go

```diff
@@ -44,6 +45,12 @@ func (fp *FakePublisher) SendAuthStatus(thingID string, errMsg *string) error {
 	return args.Error(0)
 }
 
+// SendUpdateData provides a mock function to send a update data command
```

Comment:
"provides a mock function to send **an** update data command"

Reply:
Done!